src/main/webapp/styles/homestyle.css

```diff
@@ -0,0 +1,48 @@
+/* Navigation adjustments for home page */
+#header {
+  clear: both;
+}
+
+#content {
+  width: 100%;
+  margin: 0 auto;
+  display: flex;
+}
+
+#content-left {
+  flex: 1;
+  width: 50%;
+}
+
+#content-left h2 {
+  font-size: 200%;
+  font-weight: normal;
+}
+
+#content-right {
+  flex: 1;
+  width: 50%;
```

Comment:
We have a similar two column design with a header for most of the pages might be nice to put the two column feature and the header in general.css and use that for all pages

Reply:
I had this originally but sometimes margins are different on each page so I didn't create one in the general.css. However I can create two column design template in this file that is more generic to use.

Reply:
When reviewing the other files to reflect the general two column settings some of the sizes for the fonts changes so I decided to keep it separate but the general template is in general.css.

Reply:
OK sounds good